python/examples/backend/providers/watsonx.py

```diff
@@ -16,8 +17,8 @@
 
 
 async def watsonx_from_name() -> None:
-    watsonx_llm = await WatsonxChatModel.from_name(
-        "ollama:llama3.1",
+    watsonx_llm = ChatModel.from_name(
+        "watsonx:ibm/granite-3-8b-instruct",
```

Comment:
👍🏻 

However I would use the class directly, but with the correct model.

```
ibm/granite-3-8b-instruct
```

Reply:
this is just one example showing using `ChatModel.from_name`.
in the same file there are examples using 
```
WatsonxChatModel("ibm/granite-3-8b-instruct")
```